docusaurus/docs/backends/03-casper.md

```diff
@@ -41,21 +40,9 @@ Under the hood, Odra integrates with [Casper Event Standard] and creates a few [
    
 Besides that, all the events the contract emits are registered - events schemas are written to the storage under the `__events_schema` key.
 
-:::note
-Don't forget to expose events in the module using `#[odra::module(events = [...])]`. 
```

Comment:
You have to list the events to register them in a contract, I wouldn't remove that.